### ISSUE TITLE
yolov3 tiny - evaluation on pretrained weights gives lower accuracy than expected

### ISSUE THREAD

**Issue by chenhayat** [zzh8829/yolov3-tf2#391]
According to https://pjreddie.com/darknet/yolo/ website accuracy on pre-trained weights using Coco dataset should be 23.7 mAP:
Model       | Train                | Test        | mAP | FLOPS   | FPS | Cfg | Weights
Tiny YOLO | COCO trainval | test-dev | 23.7  | 5.41 Bn | 244 | cfg | weights

However when I evaluate it on Coco 2017 with 2000 images I get: 16.1 mAP.
The weird thing is that when I reduce the number of images of evaluation the accuracy becomes higher (before evaluation I shuffle the images so there is no constant order).
Did anyone try to evaluate it  with Coco and got expected results of ~23 mAP ?
- I also reduced iou_threshold from 0.5 to 0.3 and 0.1 but this did not change anything.
